docs/specs/parsing.md

```diff
@@ -13,18 +18,75 @@
     - `DateTime`, `DateTimeOffset`
```

Comment:
Also, it might be worth stating that single, double, and decimal are lower priority.